sys-libs/compiler-rt-sanitizers/compiler-rt-sanitizers-11.0.0.ebuild

```diff
@@ -110,8 +114,13 @@ src_configure() {
 
 	if use prefix && [[ "${CHOST}" == *-darwin* ]] ; then
 		mycmakeargs+=(
-			# disable use of SDK for the system itself
-			-DDARWIN_macosx_CACHED_SYSROOT=/
+			# setting -isysroot is disabled with compiler-rt-prefix-paths.patch
+			# this allows adding arm64 support using SDK in EPREFIX
+			-DDARWIN_macosx_CACHED_SYSROOT="${EPREFIX}/MacOSX.sdk"
+			# Set this based on the SDK in EPREFIX
+			-DDARWIN_macosx_OVERRIDE_SDK_VERSION="$(realpath ${EPREFIX}/MacOSX.sdk | sed -e 's/.*MacOSX\(.*\)\.sdk/\1/')"
```

Comment:
ditto

Reply:
The same sources are used here, so ditto for my comments :)

Reply:
I updated the comment to clarify what this is used for in the build.